main.go

```diff
@@ -88,6 +89,11 @@ func main() {
 
 	ctrl.SetLogger(zap.New(zap.UseFlagOptions(&opts)))
 
+	restConfig := ctrl.GetConfigOrDie()
```

Comment:
This is now duplicate of `ctrl.GetConfigOrDie()` added in https://github.com/openshift/oadp-operator/pull/1274/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R110

Reply:
Re-based and Fixed